controller/lib/core/sensors.cpp

```diff
@@ -107,3 +141,15 @@ float get_pressure_reading(AnalogPinId pinId) {
   // Sensitivity of all pressure sensors is 1 V/kPa; no division needed.
   return ((float)runningSum * ADC_LSB);
 }
+
+float convert_diff_pressure_to_volumetric_flow(
+    float diffPressureInKiloPascals) {
```

Comment:
"Diff pressure" sounds to me like a verb phrase; perhaps "pressure diff" instead?  So something like `pressure_diff_to_flow` and `pressureDiff_kPA`?

(Also, would it be more idiomatic to call it a pressure *delta* rather than a "diff"?  I feel like diff is the CS term and delta is the engineering term, but I'm outside my depth.)

Reply:
Renamed to pressure_delta_to_volumetric_flow.